src/components/App.tsx

```diff
@@ -987,8 +987,10 @@ class App extends React.Component<AppProps, AppState> {
     window.addEventListener(EVENT.RESIZE, this.onResize, false);
     window.addEventListener(EVENT.UNLOAD, this.onUnload, false);
     window.addEventListener(EVENT.BLUR, this.onBlur, false);
-    window.addEventListener(EVENT.DRAG_OVER, this.disableEvent, false);
-    window.addEventListener(EVENT.DROP, this.disableEvent, false);
+    if (this.props.handleKeyboardGlobally) {
+      window.addEventListener(EVENT.DRAG_OVER, this.disableEvent, false);
+      window.addEventListener(EVENT.DROP, this.disableEvent, false);
+    }
```

Comment:
We might just want to scope it down to `excalidraw-container` unless we want to support globally optionally ?
cc @dwelle

Reply:
You mean like this?

```typescript
this.excalidrawContainerValue.container?.addEventListener(EVENT.DRAG_OVER, this.disableEvent, false);
this.excalidrawContainerValue.container?.addEventListener(EVENT.DROP, this.disableEvent, false);
```